### ISSUE TITLE
WCAG contrast errors

### ISSUE THREAD

**Issue by adelevie** [18F/accessibility#56]
<img width="294" alt="screen shot 2015-07-30 at 4 34 55 pm" src="https://cloud.githubusercontent.com/assets/86790/8994399/fea3c970-36d8-11e5-919c-9d90f1bd2721.png">

**Comment by mgifford** [comment]
This is difficult to see in the example above, but pulled up one version that I found with the Firefox extension WCAG color contrast checker:

<img width="1231" alt="screenshot of image with color contrast highlighted in footer" src="https://cloud.githubusercontent.com/assets/116832/13237510/b03c2394-d99a-11e5-9a38-25ad56b5eb4d.png">

Not sure where the others appear.

**Comment by nickbristow** [comment]
I've updated the colors, this should be good now.

**Comment by adelevie** [comment]
Nick, can you link the PR or commit that solved this?

**Comment by nickbristow** [comment]
@adelevie yeah the styles are controlled on the guides template which gets its styles from the ruby gem
https://github.com/18F/guides-template/pull/79
